packages/@ember/object/computed.ts

```diff
@@ -1,5 +1,13 @@
 export { ComputedProperty as default, expandProperties, alias } from '@ember/-internals/metal';
 
+// import { DeprecationOptions } from '@ember/debug';
+
+// export { alias, expandProperties } from '@ember/-internals/metal';
+
+// export function or(dependentKey: string, ...additionalDependentKeys: string[]): PropertyDecorator;
+
+// export function reads(dependentKey: string): PropertyDecorator;
+
```

Comment:
Looks like a mistake that these are left in but commented out?

Reply:
Yeah, they should go.